8.8/vips.modules

```diff
@@ -444,21 +397,22 @@
     </dependencies>
   </autotools>
 
-  <!-- here's one for poppler 0.62, but it fails cross-compiling with a
-       mysterious c++ link error, probably the same thing that affects 
-       current openjpeg, see below
```

Comment:
The mysterious C++ link error should be fixed now with the move to MinGW-w64 with posix threads. This adds one additional DLL (`libwinpthread-1.dll`), I hope that's OK:
https://stackoverflow.com/a/30390278/10952119